### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by pher1202**
I apologize Mary, I submitted a lot of things.  I was trying to figure things out.

### CODE REVIEW

**Comment by pher1202** [comment]
It's easier to maintain a web page with external CSS files because it makes it more organized.  I can locate errors faster if I know I have organized my work.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    <style>
+      .myParagraph {
+    color: purple;
+}
+    </style>
+  
+    </head>
+<body>
+    <h2 style="color:blue"> About Me </h2>
```
Comment (mbMosman) [inline index.html]: Generally you want your main heading (h1) at the top of the page, generally in a `<header>`.  All others on the page would be would be h2, h3, etc, subheadings.  They should work like an outline of the content.  h2's are subheadings of the h1/header, the h3's come below the h2s and are subheadings of that topic, etc.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<p> Hello, my name is Pao Her.  <br> I am a .Net Programmer student at Hennepin Technical College.  <br> This is my first programming course, and I'm enjoying it. <br> Here are some fun facts about me.   </p>
+    <div id="content">
+    <ul>
+        <li>I tore my right Anterior Crucial Ligament in 2009 playing football.</li>
+        <li>I'm an Eagle Scout, the highst rank in boy scouts.</li>
+        <li>I love camping, even in the snow!</li>
+        <li>Watermelons.  Cold watermelons.  Especially in the summer.</li>
+        </ul>
+    </div>
+    <main>
```
Comment (mbMosman) [inline index.html]: Usually your main tag will wrap everything that is not in the `<header>`, `<nav>`, or `<footer>`.

**Comment by mbMosman** [comment]
Feel free to update with changes and create a new pull-request.

**Pull request closed without merging**